src/lib.rs

```diff
@@ -148,13 +149,14 @@ impl Device {
         }
     }
 
-    pub fn set_latency_timer(&mut self, value: u8) -> Result<()> {
-        let result = unsafe { ffi::ftdi_set_latency_timer(self.context, value) };
+    pub fn set_latency_timer(&mut self, value: num::NonZeroU8) -> Result<()> {
```

Comment:
How do you expect this to be used? `device.set_latency_timer(std::NonZeroU8::new(2).unwrap())`?

Reply:
`std::num::NonZeroU8::new(2).map_err(1)`. Sadly, there's no type-level ints yet. Perhaps we could remap `0` to `1` inside the function to get the same effect (making the return value `-2` impossible)?

Reply:
So, this is a very interesting case. Specifically, we have three sources:
- libftdi [docs](https://www.intra2net.com/en/developer/libftdi/documentation/group__libftdi.html#ga3e0227b0c7aa859d2d2afb5ffa4c8466): "latency | Value between 1 and 255"
- D2xx (proprietary driver) [docs](https://www.ftdichip.com/Support/Documents/ProgramGuides/D2XX_Programmer's_Guide(FT_000071).pdf): "In the FT8U232AM and FT8U245AM devices, the receive buffer timeout that is used to flush remaining data from the receive buffer was fixed at 16 ms. In all other FTDI devices, this timeout is programmable and can be set at 1 ms intervals between 2ms and 255 ms."
- FT2232H [datasheet](https://www.ftdichip.com/Support/Documents/DataSheets/ICs/DS_FT2232H.pdf): "The default is 16ms, but it can be altered between 0ms and 255ms. At 0ms latency you get a packet transfer on every high speed microframe."

So there are two known documentation pieces from FTDI and what `libftdi1` does is something intermediate. I'm not sure what we want to do with this :-)